src_tests/test_lossy_boson_sampling_simulator.py

```diff
@@ -22,15 +22,13 @@ def setUp(self) -> None:
         # Generate permutation matrix and define initial state.
         self.initial_state = [1, 1, 1, 1, 0]
 
-        '''
```

Comment:
For some reasong this matrix was commented out as the test could not run this way.

Reply:
The reason was that I was using this script for different (outside of the project) tests. I'll do a general fix of the tests after I finish the new functionality.